src/classes/Options.ts

```diff
@@ -2295,6 +2299,10 @@ export function loadOptions(options?: Options): Options {
         groups: getOption('groups', ['103582791469033930'], jsonParseArray, incomingOptions),
         alerts: getOption('alerts', ['trade'], jsonParseArray, incomingOptions),
 
+        discordApiKey: getOption('discordApiKey', '', String, incomingOptions),
+        discordAdmin: getOption('discordAdmin', '', String, incomingOptions),
+        steamOfDiscordAdmin: getOption('steamOfDiscordAdmin', '', String, incomingOptions),
```

Comment:
Can we remove this option and instead use the Steam IDs from the bot's `admins` option?

Reply:
I still think what to do, if `ADMINS` contains more than one steamID (because we somehow need to connect admin's discord user id with his steamID). That's why I mentioned that `format of .env is subject to change`.

Gonna make MVP first, maybe I'll find an idea by then.

Reply:
I am also confused about what `steamOfDiscordAdmin` is for.

Reply:
That's because ADMINS is an array of ids, and when I was implementing this, I didn't know how to handle the case when it contains more than one ID (because bot should know steamID for each discordID in order for trading commands to work).

Is JSON supported for values in .env and ecosystem? If yes, I can suggest reformatting ADMINS to smth like:
```json
[{"steam": "123", "discord": "123" or null}, <...>]
```
Here "steam" is required, "discord" is optional. If you're going to release this on v5, I consider this variant as possible (it's major update, so there's nothing bad in asking user to edit some stuff which is usually not edited).

Reply:
Sorry for the late reply.
Hmmmm this will be somewhat complicated for new people to understand, and I am pretty sure they will struggle with setup later.

Probably limit Discord interaction with non-trading related commands only, i.e. commands that the bot will send offer will not be included, which are `!buy`, `!sell`, `!withdraw`, `!deposit`, `!withdrawMptf`, `!donateBptf`, and `!premium`.

Reply:
And how to whitelist admins in Discord then? Don't whitelist at all is bad; whitelist as I had implemented at that moment, but without binding to steamids - it will work, but doesn't look good.

I was going to provide migration tips in wiki and patchnote. If it won't help with understanding - idk what will. Rolling back is possible, but it will roll back good-working functional, which (imo) is essential to let the Discord UI replace Steam chat UI completely.

Also, options.json is completely json - and bot owners somehow manage to work with it. Is reformatting one exact parameter on major update really that bad?

Reply:
Ok proceed. Options.json is already preset, but whatever.